distributeaid/aggregated-public-information
# fix: remove prescision limits on Product.Item custom calculations

## What changed?
Remove the precision limits in the Product.Item custom field calculations. 

Follow up to #70

## How can you test this?
1. Run yarn develop to start the Strapi Admin
2. In the Content Manager, Add a new **Product.Item**
3. Add **Volume**, **Weight**, and **Value** components and fill in the required fields. (Using data from the Product Info spreadsheet will help by providing values that can be easily confirmed). Save the Item and verify that the following fields are now populated with the correctly calculated values and that the values (except currency) are not rounded:
  **Weight**
`itemWeightKg`
`countPerKg`
  **Volume**
`itemVolumeCBCM`
`countPerCBM`
  **Value**
`pricePerItemUSD`

4. Update an existing item with new values and confirm that the above calculations update appropriately.
